### DISCUSSION TITLE
Not working wallet Connect in web and mobile mode only in Chrome browser

### DISCUSSION THREAD

**Discussion by mehrshad0289** [WalletConnect/walletconnect-monorepo#3822] [Developer Support]
wallet Connect
It works on my site in all browsers, except for Chrome, but only using mobile

Works on desktop on all browsers
what is the problem؟

**Comment by glitch-txs** [comment]
Hi this issue needs more information, please go through our [discussions guide](https://github.com/orgs/WalletConnect/discussions/3457). Thanks!
